storagemarket/types/direct_data.go

```diff
@@ -32,6 +32,7 @@ type DirectDeal struct {
 
 	// InboundCARPath is the file-path where the storage provider will persist the CAR file sent by the client.
 	InboundFilePath string
+	InboundFileSize int64
```

Comment:
Is InboundFileSize being stored to the db? If so we will need to add it to the db code also